css/main.css

```diff
@@ -234,6 +234,7 @@ body {
 
 .events-box > * {
   margin: 0 10px;
+  padding: 5px;
   text-align: center;
   width: 100%;
   color: #e20665;
```

Comment:
Can we please add border radius to make the sharp edges smooth ??

Reply:
Done. Looks good now. thanks for the suggestion @rohan09-raj 🔥 
![image](https://user-images.githubusercontent.com/35677839/148351206-3337a142-a896-45c2-aa0e-c82509c38876.png)